pyproject.toml

```diff
@@ -62,44 +62,44 @@ namespaces = false
 
 [project.optional-dependencies]
 dev = [
```

Comment:
let's keep the `dev` and `build` profile deps pinned since they're primarily used in CI, not by library users.

Reply:
Done.